src/components/SearchForm.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import NamePicker from './NamePicker';
+import LifeEventPicker from './LifeEventPicker';
+import RelationshipPicker from './RelationshipPicker';
+import CategoryPicker from './CategoryPicker';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+
+export default function SearchForm(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="md">
+      <form className={classes.form} noValidate>
+        <Typography component="h1" variant="h5">
+          Search our society's records
+        </Typography>
+        <NamePicker />
+        <LifeEventPicker />
+        <RelationshipPicker />
+        <CategoryPicker />
+        <div style={{ marginTop: 20 }}>
+          <Button variant="outlined" color="primary" onClick={() => console.log('Submit')}>
+            Submit
+          </Button>
+          <Button
+            variant="outlined"
+            onClick={() => console.log('Clear')}
+            style={{ marginLeft: 20 }}
+          >
```

Comment:
There's also a `type='reset'` you can use to indicate this, though I think we need to implement the Clear functionality manually. Does the mock really have a 'Clear' button? I find those obnoxious, but if it's in the mock I guess we'd better implement it.

Reply:
Sorry, I will remove the clear button! It was not in the mock but in the https://www.familysearch.org/search/